src/components/statecomponent.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/prop-types */
+import style from './statecomponent.module.css';
+import navarrow from '../assets/right-arrow.svg';
+
+const State = (props) => {
+  const {
```

Comment:
- [ ] Also remove the eslint disable rule here too and include proptypes for the component.